lib/galaxy_test/api/test_tools.py

```diff
@@ -1691,7 +1691,7 @@ def test_map_over_with_output_format_actions(self, history_id):
     @skip_without_tool("output_action_change_format_paired")
     def test_map_over_with_nested_paired_output_format_actions(self, history_id):
         hdca_id = self.__build_nested_list(history_id)
-        inputs = {"input": {"batch": True, "values": [dict(map_over_type="paired", src="hdca", id=hdca_id)]}}
+        inputs = {"input": {"batch": True, "values": [dict(subcollection_type="paired", src="hdca", id=hdca_id)]}}
```

Comment:
As that's consumed by the API that's a breaking change, can you revert that please

Reply:
Renamed to `map_over_type` to avoid depreciation.